### ISSUE TITLE
Windows - Issues with connection

### ISSUE THREAD

**Issue by RayEarth2162** [peterdikant/xair-remote#2]
Hi there - thanks for creating this project. It's perfect for trying linking up Xtouch Mini with XR12! However, there seem to be some issues connecting under Python windows:

C:\Python27\xair>c:\Python27\python.exe xair-remote.py -l
Connected MIDI Ports:
    USB2.0-MIDI 0
    X-TOUCH MINI 1

C:\Python27\xair>c:\Python27\python.exe xair-remote.py "X-TOUCH MINI 1"
Found XR12 with firmware 1.16 on IP 192.168.1.93
Error: MIDI port X-TOUCH MINI 1 does not exist!

C:\Python27\xair>c:\Python27\python.exe xair-remote.py "X-TOUCH MINI"
Found XR12 with firmware 1.16 on IP 192.168.1.93
Error: MIDI port X-TOUCH MINI does not exist!

**Comment by peterdikant** [comment]
I'm sorry, I don't run Windows, so I can't help you with that. I am using the Mido library for MIDI functionality. Maybe it has bug with the port names on Windows.

**Comment by teatralnik** [comment]
I am a newbie in python. I have same problems but i've managed to pass this problem using 
devicename = None
default port is used in this case. X-touch must be default
I know that it is not a correct solution to a problem but this helps to move further and test the whole script
lib/midicontroller.py

```
> def __init__(self, devicename, state):
>         self.state = state
>         devicename = None
>         try:
>             self.inport = open_input(devicename)
>             self.outport = open_output(devicename)
>         except IOError:
>             print 'Error: MIDI port %s does not exist!' % devicename
>             
>             exit()
```

**Comment by teatralnik** [comment]
Also script dosn't work on win7. no midi sendback to controller. fun fact - on winxp works perfectly

**Comment by smadds** [comment]
Struggling to get this running on a freshly installed  RPi.

Tried the modification from @teatralnik, but no joy.

If I do a -l I get the following list:
`sudo ./xair-remote.py -l

Connected MIDI Ports:
    Midi Through:Midi Through Port-0 14:0
    X-TOUCH MINI:X-TOUCH MINI MIDI 1 20:0
`
I guess the Midi Through is the default midi device? But I can't see how to change this.
Without sudu I get a heap of python errors

Any ideas?

**Comment by peterdikant** [comment]
I finally had some time to look at this problem. Windows assigns different names for input and output ports. In order to make it easier to use, I will add an autodetection mechanism that will automatically find the X-Touch Mini.

**Comment by peterdikant** [comment]
Fixed in  5a9b88d37496276e88a13c43df328129c1158734. Please note, the updated readme.
